src/NServiceBus.Core/ObjectBuilder/LightInject/LightInject.g.cs

```diff
@@ -27,7 +27,7 @@
 ******************************************************************************/
 
 //Define the necessary compilation symbols
-#if NET452
+#if NET472
```

Comment:
was this required to make this compile? we should definitely update this code.

Reply:
yes, it was required to make the LightInject thing to compile. It was the simplest change.